lib/_stream_readable.js

```diff
@@ -184,10 +184,7 @@ Object.defineProperty(Readable.prototype, 'destroyed', {
   // userland will fail
   enumerable: false,
   get() {
-    if (this._readableState === undefined) {
-      return false;
-    }
-    return this._readableState.destroyed;
+    return !this._readableState || this._readableState.destroyed;
```

Comment:
Is this correct? It was returning `false` if `this._readableState` was `undefined`, now it returns `true`.